### ISSUE TITLE
Creating sub/nested fields

### ISSUE THREAD

**Issue by jacks0n** [jhedstrom/drupalextension#554]
Is it possible to specify sub-fields when creating Nodes?

I noticed I can set `moderation_state[state]` with just `moderation_state`.

I'm trying to populate an address and some Paragraph fields. E.g. `field_address[0][address][country_code]`.

**Comment by jhedstrom** [comment]
Indeed! In the case of address, check out this [example](https://github.com/jhedstrom/drupalextension/blob/master/features/field_handlers.feature#L74). However, that is functioning due to an `AddressHandler` in the Drupal Driver, so working with paragraphs may take some experimentation...
